docs/integrations.md

```diff
@@ -0,0 +1,33 @@
+---
+sidebar_position: 5
+---
+
+# Integrations
+
+## Rest API
+
+Dewy provides a RestAPI which allows it to be used from any language.
+See the [OpenAPI documentation](api/dewy-knowledge-base-api) for available methods.
+
+## OpenAPI Clients
+
+The RestAPI is defined using OpenAPI, allowing you to generate client libraries in a variety of languages.
+See the [OpenAPI documentation](api/dewy-knowledge-base-api) for available methods.
+
+### Python
+
+The OpenAPI client for Dewy is published to PyPI as [`dewy-client`](https://pypi.org/project/dewy-client/).
+
+### Javascript / Typescript
+
+The OpenAPI client for Dewy is published to NPM as [`dewy-ts`](https://www.npmjs.com/package/dewy-ts).
+
+## Integrations
```

Comment:
"Libraries" or something? This duplicates the main title.

Reply:
Changed to frameworks.